src/main/java/frc/robot/subsystems/Shuffleboard.java

```diff
@@ -0,0 +1,109 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.cameraserver.CameraServer;
+import edu.wpi.first.wpilibj.Timer;
+
+public class Shuffleboard extends SubsystemBase {
+  private final Joystick joystick = new Joystick(0);
+  private final Timer m_timer = new Timer();
+  private int SecondsLeft = 150;
+
+  private static Shuffleboard INSTANCE = new Shuffleboard();
+
+  /**
+   * Creates a new Shuffleboard.
+   */
+  private Shuffleboard() {
+    CameraServer.getInstance().startAutomaticCapture();
+    m_timer.start();
+    SmartDashboard.putNumber("Time Left", SecondsLeft);
+  }
+
+  public static Shuffleboard getINSTANCE() {
+    return INSTANCE;
+  }
+
+  @Override
+  public void periodic() {
+
+    // This method will be called once per scheduler run
+    // shift alt F
+    if (SecondsLeft > 0) {
+      SmartDashboard.putNumber("Timer", m_timer.get());
+      if (m_timer.get() > 1) {
+        m_timer.stop();
+        m_timer.reset();
+        SmartDashboard.putNumber("Time Left", SecondsLeft);
+        SecondsLeft = SecondsLeft - 1;
+        m_timer.start();
+      }
+      SmartDashboard.putBoolean("ENDGAME", SecondsLeft < 30);
+      SmartDashboard.putBoolean("GAME OVER", SecondsLeft < 1);
+
+      boolean buttonValueA; // A is 1
+      boolean buttonValueB; // B is 2
+      boolean buttonValueX; // X is 3
+      boolean buttonValueY; // Y is 4
+
+      double XvalueL = joystick.getRawAxis(0);
+      double YvalueL = joystick.getRawAxis(1);
+      double XvalueR = joystick.getRawAxis(4);
+      double YvalueR = joystick.getRawAxis(5);
+
+      // Left Joystick
```

Comment:
What about putting all of these in a function called `getLeftJoystickInfo`?

Reply:
If that's easier to understand while reading the code, then so it be

Reply:
Since I'm waiting on the drive team, I don't think it's worth changing that now. It might just get deleted anyway, but if we do end up using it, I will do those changes.